examples/mysql/mysql_status.rb

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env ruby
```

Comment:
Create `n` copies of this file, each of which is intended to capture metrics for a specific server. Change the constants at the head of each file to reflect that specific server's information.